CUSTOMIZATION.md

```diff
@@ -127,3 +127,29 @@ packages.  If you don't want to support a particular package (e.g. if
 you have no Emacs experience and don't want to be responsible for
 students who show up with Emacs as their only editor), you can comment
 out that particular `or_dependency`.
+
+## Updating the repository
+
+If for some reason you need to get changes from this repository
```

Comment:
for some reason, such as the installation instructions having become disconnected with the current lesson material, you need to get